kura/org.eclipse.kura.camel.cloud.factory/src/org/eclipse/kura/camel/cloud/factory/internal/CamelManager.java

```diff
@@ -180,7 +189,8 @@ public Tocd getDefinition() {
         final Set<String> ids = new TreeSet<>();
         try {
 
-            final Collection<ServiceReference<CamelFactory>> refs = FrameworkUtil.getBundle(CamelManager.class).getBundleContext().getServiceReferences(CamelFactory.class, null);
+            final Collection<ServiceReference<CamelFactory>> refs = FrameworkUtil.getBundle(CamelManager.class)
```

Comment:
Which is not present in the release-2.1.0 branch :wink: 

So I do think that for 3.0 we should move the static methods from the `ServiceLocator` class to the new `ServiceUtils` class.